requirements.txt

```diff
@@ -17,7 +17,6 @@ flask-babel==0.11.2
 flask==0.12.4
 formencode==1.3.1         # via pylons
 funcsigs==1.0.2           # via beaker
-html5lib==1.0.1           # via bleach
```

Comment:
This change (like other changes) to requirements.txt was made by `pip-compile`. html5lib is no longer an explicit dependency because it is now included as part of the bleach package - see https://github.com/mozilla/bleach/issues/386